README.md

```diff
@@ -0,0 +1,23 @@
+# Farm Directory API
+
+🐮 Serves up data about farms in the US with on-site markets
+
+# Getting Started
+
+Make sure you have `$ npm install nodemon -g` and [mongodb](https://www.mongodb.com/collateral/mongodb-3-2-whats-new?jmp=search&utm_source=google&utm_campaign={campaign}&utm_keyword=download%20mongo&utm_device=c&utm_network=g&utm_medium=cpc&utm_creative=112603790683&utm_matchtype=e&gclid=CPa3oJKmt88CFURahgod1FkLlA)
```

Comment:
Give [babel-watch](https://github.com/kmagiera/babel-watch) a try instead of nodemon. I've found its typically faster